docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -163,13 +163,160 @@ Upgrading from Chef Backend 1.x to Chef Backend 2.x requires full cluster downti
 
 ## Chef Backend 2.x to 3.x Upgrade
 
-| Running Version | Upgrade To Version | Upgrade Type |
-|---------|---------|------|
-| 2.x | 2.x | Rolling |
-| 2.1 | 3.0 | Rolling |
-| 2.0 | 3.0 | Downtime |
+### Planning
+
+The versions of Chef Backend, Elasticsearch, and Chef Infra Server running your system determines the type of upgrade, index treatment, and steps for your upgrade.
+
+#### Upgrade Type
+
+Systems with Elasticsearch 5.4.1 or lower require downtime. Systems with Elasticsearch 5.6.16 or higher use a rolling upgrade.
+
+| Running Backend Version | Upgrade To Version | Elasticsearch Version | Upgrade Type |
+|---------|---------|------|------|
+| 2.1 | 3.0 | 5.4.1 or lower | Downtime |
+| 2.1 | 3.0 | 5.6.16 or greater | Rolling |
+
+Find the version of Elasticsearch on your system with the command:
+
+```bash
+curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human
+```
+
+#### Index Treatment
+
+Chef Backend clusters that were originally installed with 2.3.0 or earlier _and then_ upgraded to version 2.3.1 will need to delete and reindex the Elasticsearch after upgrading. Chef Backend clusters first installed with version 2.3.1 or later will automatically re-index.
```

Comment:
I think it would also be helpful to maintain an index of when a particular release went out along with the release notes (but that is for some other day :))

Reply:
Thanks! That's exactly the information that I needed!

Reply:
Also, we have the Backend Release notes in the docs, although they don't have the dates attached. I put a table at the top of this document.